client/me/help/help-contact/index.jsx

```diff
@@ -376,6 +377,10 @@ const HelpContact = React.createClass( {
 			return <HelpContactConfirmation { ...confirmation } />;
 		}
 
+		if ( olark.isOlarkClosed ) {
```

Comment:
The problem here is a confusing variable name. It'd make more sense if this was `olark.isSupportClosed`